Standards/scs-0003-v1-sovereign-cloud-standards-yaml.md

```diff
@@ -115,18 +115,34 @@ then a certificate of that prerequisite scope has to be presented before the cer
 | --------------- | ------------- | ------------------------------------------------------------------------------- | ------------ |
 | `version`       | String        | Mandatory: Version of the particular list of standards                          | _v3_         |
 | `standards`     | Array of maps | Mandatory: List of standard descriptors for this particular layer               |              |
-| `stabilized_at` | Date          | ISO formatted date indicating the date after this version is considered stable. | _2022-11-09_ |
-| `deprecated_at`  | Date          | ISO formatted date indicating the date on which this version is expired.        | _2023-04-09_ |
+| `stabilized_at` | Date          | (ISO formatted) from then on, this version is considered stable (see remarks)   | _2022-11-09_ |
+| `obsoleted_at`  | Date          | (ISO formatted) from then on, this version is considered obsolete (see remarks) | _2023-03-01_ |
```

Comment:
Standard [scs-0001-v1-sovereign-cloud-standards](https://github.com/SovereignCloudStack/standards/blob/issue/591/Standards/scs-0001-v1-sovereign-cloud-standards.md) does not mention a state, called "obsolete". Instead, standard mentions "Draft", "Stable" and "Deprecated". It it makes sense, to introduce "obsolete" in [scs-0001-v1-sovereign-cloud-standards](https://github.com/SovereignCloudStack/standards/blob/issue/591/Standards/scs-0001-v1-sovereign-cloud-standards.md) in favor of consistency.

Reply:
I strongly disagree. This is a purely formalistic argument. There is (as of now) just no need for this additional state in scs-0001, it would make matters unnecessarily more complicated for no practical gain, and we have already decided against it in SIG Std/Cert.

Reply:
Reading [scs-0001-v1-sovereign-cloud-standards](https://github.com/SovereignCloudStack/standards/blob/issue/591/Standards/scs-0001-v1-sovereign-cloud-standards.md) more carefully, I figured out, that we already have an obsolete state. scs-0001-v1-sovereign-cloud-standards](https://github.com/SovereignCloudStack/standards/blob/issue/591/Standards/scs-0001-v1-sovereign-cloud-standards.md) says:

> Should a document become obsolete, it can be deprecated.
>
> Obsoletions SHOULD be announced ahead of their execution by setting the deprecated_at field to a future date and moving the status to Deprecated. This signals current and future implementors that the subject matter of the document is not considered necessary or state of the art anymore.

This means, the definition of obsolete and deprecated is a follows as:

- obsolete (implicitly defined):  A standard is obsolete, if there is a deprecated date that is after the current date
- deprecated (implicitly defined): A standard is deprecated, if there is a deprecated state that is equals or before the current date.

So I agree, we do not need an explicitly obsolete **state** and I really doubt, if we need an 'obsolete' **date** in SCS Certification YAML at all, as it is already given implicitly, with a deprecated date in the future. Instead of introduction of an obsolete **date**, I suggest to define the meaning of a future, present and past deprecated date more precisely.

Reply:
I am sorry, but that is not true.

First of all, scs-0001 and scs-0003 are independent of one another. Of course, we nevertheless try not to introduce confusing differences.

Second, scs-0001 does not have two states, it merely has bad wording. Hannes changed `obsolete` to `deprecated` and he seems to have missed a few occurrences.